src/main/java/com/mgmtp/radio/support/validator/user/CreateFavoriteSongValidator.java

```diff
@@ -17,11 +18,13 @@
 	private final FavoriteSongService favoriteSongService;
 	private final UserService userService;
 	private final MessageSource messageSource;
+	private final SongService songService;
 
-	public CreateFavoriteSongValidator(FavoriteSongService favoriteSongService, UserService userService, MessageSource messageSource) {
+	public CreateFavoriteSongValidator(FavoriteSongService favoriteSongService, UserService userService, MessageSource messageSource, SongService songService) {
 		this.favoriteSongService = favoriteSongService;
 		this.userService = userService;
 		this.messageSource = messageSource;
```

Comment:
Change MessageSource to MessageSourceAccessor, MessageSourceAccessor will auto handle locale.

Reply:
if we want to use MessageSourceAccessor, we have to create a bean for it in config file.
should we add 1 more step?

Reply:
Both new and create bean is OK, your choice? :D